### PR TITLE
Creating the best file ever

### PR DESCRIPTION
**Pull request by richardbadman**
This will allow me to get all the approvals

### CODE REVIEW

File: kam-sennedy.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+def main():
+    peers = Approval(['Bradley', 'Sam', 'Sergey', 'Yohan', 'Olivia', 'Chaice', 'Olga'])
+    for peer in peers:
+        peers.approve(peer, True)
+
+    print('Thanks guys!')
+
+
+if __name__ == '__main__':
+    main()
```
Comment (SamJamesKennedy) [inline kam-sennedy.py:23]: No newline at end of file

**Review by SamJamesKennedy — CHANGES_REQUESTED** [review]

File: kam-sennedy.py (hunk 1 above)
Comment (richardbadman) [reply kam-sennedy.py:23]: You're not meant to dick

**Review by richardbadman — COMMENTED** [review]
LGTM

**Review by olga-brandwatch — APPROVED** [review]

**Review by SergeEgorov — APPROVED** [review]

**Comment by bw-matthew** [comment]
This PR only got 2 approvals which is less than the 4 that https://github.com/bw-matthew/i-hope-i-remember-to-make-this/pull/12 got

Also you didn't order the peers alphabetically

**Pull request closed without merging**